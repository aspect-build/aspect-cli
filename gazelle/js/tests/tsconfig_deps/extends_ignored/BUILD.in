# gazelle:js_ignore_imports bad
