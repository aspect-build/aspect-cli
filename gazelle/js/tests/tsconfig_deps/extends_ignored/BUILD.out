load("@aspect_rules_ts//ts:defs.bzl", "ts_config")

# gazelle:js_ignore_imports bad

ts_config(
    name = "tsconfig",
    src = "tsconfig.json",
)
